examples/aei_from_events/aei_from_events.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+
+all_cat = ['festivals', 'school-holidays', 'airport-delays', 'sports',
+           'performing-arts', 'concerts', 'public-holidays', 'community',
+           'observances', 'severe-weather', 'expos', 'conferences',
+           'health-warnings', 'academic', 'disasters', 'politics', 'terror']
+
+nonattended = ['public-holidays', 'observances', 'school-holidays', 'academic']
+
+attended = ['festivals', 'sports', 'performing-arts',
+            'concerts', 'community', 'expos', 'conferences']
+
+unscheduled = ['airport-delays', 'health-warnings', 'disasters', 'politics', 'terror', 'severe-weather']
+
```

Comment:
Please use uppercase for constants, e.g. ALL_CAT. Also, as it's a public repository, I'd suggest to avoid abbreviations. `ALL_CATEGORIES`.

Reply:
Can do! This is easy to fix.

Reply:
I have done so for each of the corresponding event categories, but have placed `ALL_CATEGORIES` at the bottom since it is representative of the sum of the distinct event categories.